.github/workflows/python-tests.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           - os: ubuntu-20.04
             python-version: "3.7"
           - os: ubuntu-20.04
-            python-version: "3.10"
+            python-version: "3.11"
           - os: macos-11
             python-version: "3.10"
```

Comment:
I would also update the Python version on macos & windows

Reply:
Fine for me. But we leave Python 3.10 for all the other CI tasks? lint, docs and wheels

Reply:
Then lets bump all of them to be consistent and avoid probable weird error messages.

However, torch doesn't still not support `3.11`, right?

Reply:
> However, torch doesn't still not support 3.11, right?

What makes you think so? They have wheels for cp311 both here https://download.pytorch.org/whl/torch/ and on on PyPI: https://pypi.org/project/torch/#files

Reply:
Oh, this: https://github.com/lab-cosmo/equistore/actions/runs/5655781307/job/15321547016#step:8:169

We run some of the tests against torch 1.11, which does not support Python 3.11. We could do a Python version in the matrix as well for these